### PR TITLE
Chapter 4 - 2022 update

### PR DESCRIPTION
**Pull request by icanzilb**

### CODE REVIEW

File: 04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -278,6 +278,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		795EF16326E4D0C20074D4C2 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```
Comment (icanzilb) [inline 04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj:281]: @jrturton what is this setting 🤔

Comment (jrturton) [reply 04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj:281]: Without it the swiftlint phase causes a compiler warning. It’s new in xcode 14

Comment (icanzilb) [reply 04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj:281]: Oh, gotcha yes, I see ...

Comment (icanzilb) [reply 04-custom-asynchronous-sequences-with-asyncstream/projects/starter/Blabber.xcodeproj/project.pbxproj:281]: Thanks

**Pull request merged**